Tests/AnnotatedMock.php

```diff
@@ -52,12 +52,21 @@ class AnnotatedMock
     /**
      * The quux property.
      *
-     * @GDPR\Export()
+     * @GDPR\Export(fieldName="quuxs")
```

Comment:
My preference for the name for this property would be "alias" as the property name is used as name otherwise.